doc/development/plugins.rst

```diff
@@ -130,7 +130,9 @@ Defining the ``__init__.py`` method of a custom device is not necessary; by defa
 the :class:`~.QubitDevice` initialization will be called, where the user can pass the
 following arguments:
 
-* ``wires`` (*int*): the number of wires on the device.
+* ``wires`` (*int* or *Iterable[Number, str]*): The number of subsystems represented by the device,
+  or iterable that contains unique labels for the subsystems as numbers (i.e., ``[-1, 0, 2]``)
+  or strings (``['ancilla', 'q1', 'q2']``).
```

Comment:
Is it definitely numbers *or* strings (i.e. not Union[Number, str])?

Reply:
```suggestion
  and/or strings (``['ancilla', 'q1', 'q2']``).
```

Reply:
You are right...do you feel it it wrong otherwise? May need to add this to all the device's docstring then :)